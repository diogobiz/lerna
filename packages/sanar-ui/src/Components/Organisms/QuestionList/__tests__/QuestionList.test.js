
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESQuestionList from '../'

it('renders correctly', () => {
    const component = (
        <ESQuestionList>
            <h1>ESQuestionList</h1>
        </ESQuestionList>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
