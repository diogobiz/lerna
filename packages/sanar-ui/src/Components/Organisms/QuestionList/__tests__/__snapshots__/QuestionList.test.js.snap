// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-card es-card es-question-list ant-card-bordered ant-card-small"
>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <h1>
      ESQuestionList
    </h1>
  </div>
</div>
`;
